src/components/Link.tsx

```diff
@@ -222,4 +224,17 @@ const Link: React.FC<IProps> = ({
   )
 }
 
+export function navigate(
```

Comment:
New plugin doesn't provide this function. Implemented a similar approach as the previous one was doing https://github.com/wiziple/gatsby-plugin-intl/blob/master/src/link.js#L42